graphics/fontconfig/Portfile

```
@@ -64,7 +64,7 @@ pre-configure {
 
     # Fix building for older macOS versions via MACOSX_DEPLOYMENT_TARGET
     # https://bugs.freedesktop.org/show_bug.cgi?id=102986
-    if {${os.platform} eq "darwin" && [vercmp ${macosx_version} >= 10.12]} {
+    if {${os.platform} eq "darwin" && [vercmp ${macos_version} >= 10.12]} {
```

Take special care every time you want to propose such a change since the two variables have different values. `macosx_version` is only the major version, like `10.15` or `11`, while `macos_version` is the full version, like `10.15.7` or `11.7.10`. Swapping these variables carelessly can result in unintentionally changed behavior.

We could make this change here since it changes nothing here, but my preference might be not to change it, since it ain't broke, and since we have no need here to check the full OS version; we're only interested in the major version.

```suggestion
    if {${os.platform} eq "darwin" && [vercmp ${macosx_version} >= 10.12]} {
```

---

Sure. Just flagging, `port lint --nitpick` suggests replacing the old var with the new one